### PR TITLE
some minor frontend text improvements

### PR DESCRIPTION
**Pull request by JustIanJ**
Please comment on this pull request if you have any questions or if you want something changed back.

### CODE REVIEW

File: cogs/event_handler.py [hunk 1]
```diff
@@ -95,8 +95,8 @@ async def on_message(self, message: nextcord.Message):
                         )
                         embed = nextcord.Embed(color=0x0DD91A)
                         embed.add_field(
-                            name=f"You ({user.display_name}) have been muted",
-                            value="You have been muted for 10 minutes.\nIf you think this was a mistake, please contact an owner or admin",
+                            name=f"You, ({user.display_name}), have been muted",
```
Comment (codeman1o1) [inline cogs/event_handler.py:98]: I think the parentheses can be removed

File: cogs/event_handler.py [hunk 2]
```diff
@@ -210,7 +210,7 @@ async def on_member_remove(self, member: nextcord.Member):
         with suppress(nextcord.errors.HTTPException):
             embed = nextcord.Embed(
                 color=0x0DD91A,
-                title=f"Bye {member.display_name} :wave:\nIt was great having you!",
+                title=f"Bye {member.display_name} :wave:\nIt was great having you! We hope to see you back soon.",
```
Comment (codeman1o1) [inline cogs/event_handler.py:213]: It would be nice to have the extra text on a new line

File: cogs/event_handler.py [hunk 3]
```diff
@@ -95,8 +95,8 @@ async def on_message(self, message: nextcord.Message):
                         )
                         embed = nextcord.Embed(color=0x0DD91A)
                         embed.add_field(
-                            name=f"You ({user.display_name}) have been muted",
-                            value="You have been muted for 10 minutes.\nIf you think this was a mistake, please contact an owner or admin",
+                            name=f"You, ({user.display_name}), have been muted",
+                            value="You have been muted for 10 minutes.\nIf you think the bot has made a mistake, please contact an owner or admin",
```
Comment (codeman1o1) [inline cogs/event_handler.py:99]: I don't think this changes makes it better, so please revert it

**Review by codeman1o1 — CHANGES_REQUESTED** [review]

File: cogs/event_handler.py (hunk 2 above)
Comment (JustIanJ) [reply cogs/event_handler.py:213]: how do i do that?

Comment (JustIanJ) [reply cogs/event_handler.py:213]: @codeman1o1

File: cogs/event_handler.py (hunk 3 above)
Comment (codeman1o1) [reply cogs/event_handler.py:99]: The "the bot has made" addition

**Review by codeman1o1 — APPROVED** [review]

**Pull request merged**